### PR TITLE
The Hypospray Belt

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Recent concerns from some corpsmen want to base themselves around the hypospray but is in short supply of them. The Research and Development team provides! Now, they get a hypospray belt dedicated to holding reagent bottles and hyposprays to their heart's content! It's up to them if they want to use it or not.

Hypospray belt can only hold analyser, hyposprays, reagent bottles, and reagent bottle cases.

Corpsmen cannot have preloaded hyposprays consisting of bicaridine, kelotane, tramadol, dexalin, inaprovaline, peridaxon, and quickclot YET! I just need to do some magic about that.

## Why It's Good For The Game

JPR approves of this. He is the residental medical balance person.

Bravemole gives better gameplay choices and design on hypospray.

Many good corpmen carry multiple hypospray since they can use this for marines that are running away from the corpsmen. Using autoinjectors as corpmen is kinda wack.

I swear, this won't be like how corpsmen have big hypospray for free. I might get yelled at by MJP since #5700.

Also, impending #9236 .

## Changelog
:cl:
add: the hypospray belt! The belt is loaded with bicaridine, kelotane, tramadol, dexalin, tricordrazine, inaprovaline, spaceacillin, hypervene, peridaxon, and quickclot hypospray along with reagent bottles.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -234,6 +234,44 @@
...
+	desc = "The M276 is the standard load-bearing equipment of the TGMC. It consists of a modular belt with various clips. This version is a less common configuration, designed to transport hyposprays and reagent_containers."
+	icon_state = "hypospraybelt"
+	item_state = "medicbag"
+	storage_slots = 21
+	max_storage_space = 42
+	max_w_class = 2
+	can_hold = list(
+		/obj/item/healthanalyzer,
+		/obj/item/reagent_containers/glass/bottle,
+		/obj/item/reagent_containers/hypospray/autoinjector,
```
Comment (Lumipharon) [inline code/game/objects/items/storage/belt.dm:248]: Doesn't this mean you can't actually put hypos in it?
```suggestion
		/obj/item/reagent_containers/hypospray/,
```

**Comment by Lumipharon** [comment]
Definitely cool to have, but I do wonder how useful it is in practice, as you can get the same outcome more efficiently without this belt.

The reagent bottles are only 60u, where as a pill bottle holds up to 240u (or 480u, in the case of inaprov).
Because you can just add pills to a hypo as easily as using a reagent bottle, if I wanted to replicate a hypo belt I could have a medkit with the 7 hypos you've got in the belt (excluding inaprov, that's almost always used on poor sods that can't run away), then fill the normal medic belt with a hugely more expansive amount of all those reagents, as well as all the other junk medics need.

For the hypo belt medic, they'll have less reagents, and will still need to devote storage space else where to all the non reagent junk they need to carry.

**Comment by novaepee** [comment]
MJP and others told me to put in reagent bottle cases that hold 3 reagent bottles inside of them. I won't be surprise if storage creep, but MJP knows storage creep well, so I trust his word on this.

**Comment by novaepee** [comment]
image

File: code/game/objects/items/storage/firstaid.dm [hunk 2]
```diff
@@ -262,6 +262,68 @@
 	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
 
 
+	/*
+* Bottle Cases
+*/
+
+
+/obj/item/storage/bottlecase
```
Comment (BraveMole) [inline code/game/objects/items/storage/firstaid.dm:270]: why does this exist ?

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Review by BraveMole — APPROVED** [review]

**Comment by novaepee** [comment]
After getting feedback, corpsmen want additional types of hypospray and a dex bottle case.

File: code/game/objects/items/storage/belt.dm [hunk 3]
```diff
@@ -256,13 +256,17 @@
...
 	new /obj/item/storage/bottlecase/tram(src)
 	new /obj/item/reagent_containers/hypospray/advanced/tricordrazine(src)
 	new /obj/item/storage/bottlecase/tricord(src)
 	new /obj/item/reagent_containers/hypospray/advanced/dexalin(src)
 	new /obj/item/reagent_containers/glass/bottle/dexalin(src)
 	new /obj/item/reagent_containers/hypospray/advanced/inaprovaline(src)
+	new /obj/item/reagent_containers/hypospray/advanced/spaceacillin(src)
+	new /obj/item/reagent_containers/hypospray/advanced/hypervene(src)
 	new /obj/item/reagent_containers/hypospray/advanced/peridaxon(src)
 	new /obj/item/reagent_containers/hypospray/advanced/quickclot(src)
```
Comment (BraveMole) [inline code/game/objects/items/storage/belt.dm:271]: all the single hypospray should not be hyposray, but injectors. This should definitly not procure more reagants than normal medbelt

File: code/game/objects/items/storage/belt.dm [hunk 4]
```diff
@@ -256,13 +256,17 @@
 	new /obj/item/storage/bottlecase/bica(src)
 	new /obj/item/reagent_containers/hypospray/advanced/kelotane(src)
 	new /obj/item/storage/bottlecase/kelo(src)
+	new /obj/item/reagent_containers/hypospray/advanced/dylovene(src)
+	new /obj/item/storage/bottlecase/dylo(src)
 	new /obj/item/reagent_containers/hypospray/advanced/tramadol(src)
 	new /obj/item/storage/bottlecase/tram(src)
 	new /obj/item/reagent_containers/hypospray/advanced/tricordrazine(src)
```
Comment (BraveMole) [inline code/game/objects/items/storage/belt.dm:263]: no other belts procure tricordrazine like that free

**Review by BraveMole — CHANGES_REQUESTED** [review]
I'm freezing this pr until mjp finishes medical rebalance

**Review by Kuro020 — APPROVED** [review]

**Review by BraveMole — APPROVED** [review]

**Pull request merged**